### PR TITLE
[zh-cn]sync docs/contribute

### PR DESCRIPTION
**Pull request by my-git9**
Fix https://github.com/kubernetes/website/issues/39984

### CODE REVIEW

File: content/zh-cn/docs/contribute/style/hugo-shortcodes/index.md [hunk 1]
```diff
@@ -448,6 +448,62 @@ Renders to:
...
+<!--
+### Source code files
+You can use the `{{</* codenew */>}}` shortcode to embed the contents of file
+in a code block to allow users to download or copy its content to their clipboard.
+This shortcode is used when the contents of the sample file is generic and reusable, 
+and you want the users to try it out themselves.
+-->
+当示例文件的内容是通用的、可复用的，并且希望用户自己尝试使用此短代码时。
+你可以使用 `{{</* codenew */>}}` 短代码将文件内容嵌入代码块中，
+以允许用户下载或复制其内容到他们的剪贴板。
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/style/hugo-shortcodes/index.md:460]: 标点符号和语序都不对

**Comment by my-git9** [comment]
@tengqm I added syncing of more files in this commit for the issue:https://github.com/kubernetes/website/issues/39984

File: content/zh-cn/docs/contribute/localization.md [hunk 2]
```diff
@@ -155,6 +164,20 @@ You'll need to know the two-letter language code for your language. Consult the
...
 two-letter code for Korean is `ko`.
 
+If the language you are starting a localization for is spoken in various places
+with significant differences between the variants, it might make sense to
+combine the lowercased ISO-3166 country code with the language two-letter code.
+For example, Brazilian Portuguese is localized as `pt-br`.
+-->
+你需要知道你的语言的两个字母的语言代码。
+请查阅 [ISO 639-1 标准](https://www.loc.gov/standards/iso639-2/php/code_list.php)
+以查找你的本地化的两字母语言代码。例如，韩语的两字母代码是`ko`。
```
Comment (ydFu) [inline content/zh-cn/docs/contribute/localization.md:174]:
```suggestion
以查找你的本地化的两字母语言代码。例如，韩语的两字母代码是 `ko`。
```

**Review by ydFu — COMMENTED** [review]
This paragraph looks good to me, just some minor modifications needed.

**Review by ydFu — APPROVED** [review]
/lgtm

**Comment by windsonsea** [comment]
CI is failed temporarily. Run the following commands to reset the CI flow:
```
git reset
git commit --amend --no-edit
git push -f
```

**Pull request merged**